appveyor.yml

```diff
@@ -64,11 +70,17 @@ install:
     - cmd: rmdir C:\cygwin /s /q
 
     # Use the pre-installed Miniconda for the desired arch
-    - ps: if($env:TARGET_ARCH -eq 'x86')
-            {$root = "C:\Miniconda"}
-          else
-            {$root = "C:\Miniconda-x64"}
-          $env:path="$root;$root\Scripts;$root\Library\bin;$($env:path)"
+    #
+    # However, it is really old. So, we need to update some
+    # things before we proceed. That seems to require it being
+    # on the path. So, we temporarily put conda on the path
+    # so that we can update it. Then we remove it so that
+    # we can do a proper activation.
+    - cmd: set "OLDPATH=%PATH%"
+    - cmd: set "PATH=%CONDA_INSTALL_LOCN%\\Scripts;%CONDA_INSTALL_LOCN%\\Library\\bin;%PATH%"
+    - cmd: conda update --yes --quiet conda
```

Comment:
AppVeyor [balked]( https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.4157 ) at my [syntax]( https://github.com/conda-forge/staged-recipes/commit/d06dd0d02db1b07cd7b064808f9bb05bc16a4c20#diff-180360612c6b8c4ed830919bbb4dd459R73 ) when I tried that. Maybe it was getting confused with the `\`s.